From 824bb1eecbef4948246722470fa70dfbf58b8bc7 Mon Sep 17 00:00:00 2001
From: "Eduardo Lima (Etrunko)" <etrunko@redhat.com>
Date: Wed, 29 Jun 2016 16:40:36 -0300
Subject: [PATCH] Update timer to refresh ovirt foreign menu

This is a temporary solution, as discussed in the bug. We will adjust
the timer to refresh the ISO list from 15 seconds to 5 minutes (300
seconds), while reworking in the UI to replace the menu with a dialog,
which seems a saner way to display the list.

Resolves: rhbz#1347726

Signed-off-by: Eduardo Lima (Etrunko) <etrunko@redhat.com>
---
 src/ovirt-foreign-menu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ovirt-foreign-menu.c b/src/ovirt-foreign-menu.c
index cd5fdc6..e66081c 100644
--- a/src/ovirt-foreign-menu.c
+++ b/src/ovirt-foreign-menu.c
@@ -797,7 +797,7 @@ static void iso_list_fetched_cb(GObject *source_object,
     ovirt_foreign_menu_set_files(OVIRT_FOREIGN_MENU(user_data), files);
     g_list_free(files);
 
-    g_timeout_add_seconds(15, ovirt_foreign_menu_refresh_iso_list, user_data);
+    g_timeout_add_seconds(300, ovirt_foreign_menu_refresh_iso_list, user_data);
 }
 
 
